charts/nginz/templates/secret.yaml

```
@@ -15,5 +15,7 @@ data:
   {{- with .Values.secrets }}
   zauth.conf: {{ .zAuth.publicKeys | b64enc | quote }}
   basic_auth.txt: {{ .basicAuth | b64enc | quote }}
+  {{- if .oAuth.publicKeys }}
```

this will still fail if `.oAuth` doesn't exist.

```suggestion
  {{- if .oAuth }}
```